package.json

```diff
@@ -99,5 +99,8 @@
     "stylelint": "^7.3.1",
     "stylelint-config-standard": "^13.0.2",
     "through": "^2.3.8"
+  },
+  "dependencies": {
+    "mousetrap": "^1.6.0"
```

Comment:
There is something wrong here as all `devDependencies` should be in `dependencies` for me... or we assume that since the final user should run `npm install` etc .. we put all our dep in devDeps ?